packages/jest-message-util/src/types.ts

```diff
@@ -1,12 +1,5 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
-import type {StackData} from 'stack-utils';
-
-export interface Frame extends StackData {
```

Comment:
This is also technically breaking since this type used to be a good bit wider. I could fix it so that it includes all the data it used to relatively easily though